library/src/main/java/com/danikula/videocache/file/Files.java

```diff
@@ -47,7 +47,7 @@ static void makeDir(File directory) throws IOException {
     static void setLastModifiedNow(File file) throws IOException {
         if (file.exists()) {
             long now = System.currentTimeMillis();
-            boolean modified = file.setLastModified(now); // on some devices (e.g. Nexus 5) doesn't work
+            boolean modified = file.setLastModified(now/1000*1000); // on some devices (e.g. Nexus 5) doesn't work
```

Comment:
@ta893115871 please explain you fix. Who and when does throw `IOException`? Why you divide now to `1000*1000`?